src/Nethermind/Nethermind.Optimism/OptimismEIP1559Parameters.cs

```diff
@@ -25,69 +29,102 @@ public EIP1559Parameters(byte version, UInt32 denominator, UInt32 elasticity)
         Elasticity = elasticity;
     }
 
-    public static bool TryCreate(byte version, UInt32 denominator, UInt32 elasticity, out EIP1559Parameters parameters, [NotNullWhen(false)] out string? error)
+    public EIP1559Parameters(byte version, UInt32 denominator, UInt32 elasticity, UInt64? minBaseFee) : this(version, denominator, elasticity)
+    {
+        MinBaseFee = minBaseFee;
+    }
+
+    public static bool TryCreateV0(UInt32 denominator, UInt32 elasticity, out EIP1559Parameters parameters, [NotNullWhen(false)] out string? error)
     {
         error = null;
         parameters = default;
 
-        if (version != 0)
+        if (denominator == 0 && elasticity != 0)
         {
-            error = $"{nameof(version)} must be 0";
+            error = $"{nameof(denominator)} cannot be 0 unless {nameof(elasticity)} is also 0";
             return false;
         }
 
-        if (denominator == 0 && elasticity != 0)
+        parameters = new EIP1559Parameters(0, denominator, elasticity);
+
+        return true;
+    }
+
+    public static bool TryCreateV1(uint denominator, uint elasticity, UInt64 minBaseFee, out EIP1559Parameters parameters,
+        [NotNullWhen(false)] out string? error
+    )
+    {
+        error = null;
+        parameters = default;
+
+        if (denominator == 0)
         {
-            error = $"{nameof(denominator)} cannot be 0 unless {nameof(elasticity)} is also 0";
+            error = $"{nameof(denominator)} cannot be 0";
             return false;
         }
 
-        parameters = new EIP1559Parameters(version, denominator, elasticity);
+        parameters = new EIP1559Parameters(1, denominator, elasticity, minBaseFee);
+
         return true;
     }
 
-    public bool IsZero() => Denominator == 0 && Elasticity == 0;
+    public bool IsZero() => Version == 0 && Denominator == 0 && Elasticity == 0 && MinBaseFee == null;
```

Comment:
`IsZero` is used to default the EIP1559 parameters in Holocene. For Jovian, the same defaulting mechanism should be used and note that `MinBaseFee` is never `null` according to the spec (otherwise it's invalid).

With this code, after Jovian, a valid instance of `EIP1559Parameters` struct will never return `false` for this method since `MinBaseFee == null` is always false, which is incorrect.